File: Beginner/Mutated Minions (CHN15A)/minions.java
import java.util.*;
import java.lang.*;
import java.io.*;


class Codechef {
	public static void main (String[] args) throws java.lang.Exception {
		Scanner scan = new Scanner(System.in);
		if(scan.hasNext()){
		    int t = scan.nextInt();
		    for(int c=0;c<t;c++){
		        int N = scan.nextInt();
		        int K = scan.nextInt();
		        int arr[]=new int[N];
		        for(int i=0;i<N;i++){
		            arr[i] = scan.nextInt();
		        
		            arr[i] += K;
		        }
		        int count=0;
		        for(int i=0; i<N; i++){
		            if(arr[i]%7==0)
		                count++;
		        }
		        System.out.println(count);
		    }
        }
        scan.close();
	}
}

File: Beginner/Mutated Minions (CHN15A)/minions.cpp
#include <iostream>
using namespace std;

int main() {
    int t;
    cin>>t;
    while(t--){
        int N,K;
        cin>>N>>K;
        int arr[N];
        for(int i=0;i<N;i++){
            cin>>arr[i];
        }
        int s=0;
        for(int i=0;i<N;i++){
            int k=arr[i]+K;
            if(k%7==0){
                s++;
            }
        }
        cout<<s<<endl;
    }
	return 0;
}